src/Microsoft.ML.AutoML/ColumnInference/TextFileContents.cs

```diff
@@ -23,14 +23,16 @@ public class ColumnSplitResult
 
             public bool AllowQuote { get; set; }
             public bool AllowSparse { get; set; }
+            public bool  ReadMultilines { get; set; }
 
-            public ColumnSplitResult(bool isSuccess, char? separator, bool allowQuote, bool allowSparse, int columnCount)
+            public ColumnSplitResult(bool isSuccess, char? separator, bool allowQuote, bool readMultilines, bool allowSparse, int columnCount)
```

Comment:
Following naming convention would be:
```suggestion
            public ColumnSplitResult(bool isSuccess, char? separator, bool allowQuote, bool allowMultiline, bool allowSparse, int columnCount)
```

Reply:
TextLoader Option uses ReadMultilines (So is allowQuote and allowSparse), and we'd better follow their naming style.

Reply:
### Resolved